### ISSUE TITLE
Add new subtitles providers 

### ISSUE THREAD

**Issue by Nomuas** [pannal/Sub-Zero.bundle#251]
Hello,

Could you consider to add any of these providers :
- sous-titres.eu
- u-sub.net
- betaseries

Thanks in advance !

**Comment by pannal** [comment]
Are those tied to a specific language? The current focus is 1.5, which will implement the new providers subliminal support. After that subscene.com hopefully will be added.

**Comment by Nomuas** [comment]
Yes it's typically the best source quality for french subtitles. 
Betaseries is an aggregator of multiples providers.

**Comment by delmicio** [comment]
tusubtitulo.com 

Best/Fastest for spanish TV Shows.

**Comment by SvenErik1968** [comment]
A possible provider for Thai subtitles would be thaisubtitle.com
